### PR TITLE
feat: status: add get_fast_commit_dep() to find out what dep can be s…

### PR DESCRIPTION
**Pull request by drmingdrmer**
## Type of change       

- **New feature**       

# Checklist:

- [x] **Self-review**: I have performed a **self-review** of my own code
- [ ] **Comment**:     I have **commented my code**, particularly in hard-to-understand areas
- [ ] **Doc**:         I have made corresponding changes to the **documentation**
- [ ] **No-warnings**: My changes generate **no new warnings**
- [x] **Add-test**:    I have added **tests** that prove my fix is effective or that my feature works
- [x] **Pass**:        New and existing **unit tests pass** locally with my changes

### CODE REVIEW

File: components/epaxos/src/replica/status.rs [hunk 1]
```diff
@@ -53,3 +54,48 @@ impl Status {
...
+
+    if n < fast_quorum {
+        return None;
+    }
+
+    let x = (n - fast_quorum) as usize;
+
+    for i in 0..=x {
+        let dep = &deps[i];
+        if dep == &deps[i + fast_quorum as usize - 1] {
```
Comment (liubaohai) [inline components/epaxos/src/replica/status.rs]: 这种判断方式是不是滑动窗口。

Comment (drmingdrmer) [reply components/epaxos/src/replica/status.rs]: 啥叫不是滑动窗口...不是咋的了

Comment (liubaohai) [reply components/epaxos/src/replica/status.rs]: 没事

Comment (drmingdrmer) [reply components/epaxos/src/replica/status.rs]: 啊对...有点类似

**Pull request merged**